gnocchi/rest/__init__.py

```diff
@@ -375,6 +375,21 @@ def MeasuresListSchema(measures):
         times.tolist(), values))
 
 
+VALID_TRANSFORMATION_METHODS = ["absolute", "negative"]
+
+
+def TransformSchema(transform):
```

Comment:
pyparsing? But I don't see any pyparsing here.

```python
TransformSchema = voluptuous.Schema([VALID_TRANSFORMATION_METHODS])

# later use
transform = TransformSchema(transform or "")
```
just an idea but it might look neater?